osf_tests/test_generate_sitemap.py

```diff
@@ -0,0 +1,164 @@
+import os
+
+import pytest
+import mock
+import shutil, tempfile, xml, urlparse
+
+from scripts import generate_sitemap
+from osf_tests import factories
```

Comment:
Rules break down when there is a norm in the codebase:
<img width="213" alt="screen shot 2017-07-31 at 10 01 16 am" src="https://user-images.githubusercontent.com/1322421/28781224-59febc32-75d7-11e7-96eb-b080799513c5.png">
<img width="215" alt="screen shot 2017-07-31 at 10 01 44 am" src="https://user-images.githubusercontent.com/1322421/28781227-5c33c272-75d7-11e7-902b-ee6295bbf2fe.png">